air/src/proof.rs

```diff
@@ -165,17 +163,22 @@ impl ProofOptions {
     pub const fn hash_fn(&self) -> HashFunction {
         self.hash_fn
     }
+
+    /// Returns the execution options specified for this [ProvingOptions]
+    pub const fn get_execution_options(&self) -> ExecutionOptions {
+        self.exec_options
+    }
```

Comment:
Should we change this to `execution_options()` and return `&ExecutionOptions` from it?